docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -584,28 +584,14 @@ oc apply -f - <<EOF
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: nova-compute-ffu
+  name: nova-compute-extraconfig
```

Comment:
this passes in CI, but I'm not sure how updating this cm works in-place as we've created it already earlier https://github.com/openstack-k8s-operators/data-plane-adoption/pull/376/files ?
I though k8s apisesrver reject updating configmaps, and those need to be recreated?

Reply:
no you can update them as often as you like.

we are not using imutable config maps.
in this specific instance it would be incorrect too as users are expected to create there services once but modify the content of the reference config maps whenever they need to modify the deployed config.